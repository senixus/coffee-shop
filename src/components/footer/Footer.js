import { Link } from "react-router-dom";
import "./footer.scss";

const Footer = () => {
  return (
    <footer>
      <div>&copy; 2021 All rights reserved</div>
      <ul>
        <li>
          <Link to="/">Home</Link>
        </li>
        <li>
          <Link to="/shop">Shop</Link>
        </li>
        <li>
          <Link to="/contact">Contact</Link>
        </li>
      </ul>
    </footer>
  );
};

export default Footer;
