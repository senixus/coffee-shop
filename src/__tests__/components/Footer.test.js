import { shallow } from "enzyme";
import React from "react";
import Footer from "../../components/footer/Footer";

it("expect to render Footer component", () => {
  const wrapper = shallow(<Footer />);
  expect(wrapper.length).toBe(1);
  expect(wrapper).toMatchSnapshot();
});
