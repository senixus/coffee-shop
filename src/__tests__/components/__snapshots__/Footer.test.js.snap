// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Footer component 1`] = `
<footer>
  <div>
    Â© 2021 All rights reserved
  </div>
  <ul>
    <li>
      <Link
        to="/"
      >
        Home
      </Link>
    </li>
    <li>
      <Link
        to="/shop"
      >
        Shop
      </Link>
    </li>
    <li>
      <Link
        to="/contact"
      >
        Contact
      </Link>
    </li>
  </ul>
</footer>
`;
